sortinghat/api.py

```diff
@@ -848,7 +878,15 @@ def move_identity(db, from_id, to_uuid):
         if fid.uuid == to_uuid:
             return
 
+        fuid = session.query(UniqueIdentity).\
+            filter(UniqueIdentity.uuid == fid.uuid).first()
+
+        last_modified = datetime.datetime.utcnow()
+
         fid.uuid = to_uuid
+        fuid.last_modified = last_modified
+        tuid.last_modified = last_modified
+        fid.last_modified = last_modified
 
 
 def match_identities(db, uuid, matcher):
```

Comment:
Oh, great! Less places in which to update the last_modified field than what I have in mind. Nice!